priv/maker-ui/en.json

```diff
@@ -198,5 +198,13 @@
     "share-poll": "Share this poll",
     "link-poll": "Link to this poll",
     "embed-poll": "Embed this poll"
+  },
+  "quiz_title": {
```

Comment:
should we do `quizzes_title` to keep consitent everywhere? the `routes.jade` looks like...
```
DefaultRoute(name='quizzes-title' handler=QuizzesTitle)
```